proto/process/agent.proto

```diff
@@ -465,3 +465,35 @@ message Host {
 message DNSEntry {
   repeated string names = 1;
 }
+
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto
+message Pod {
+	string name = 1;
+	string namespace = 2;
+	int64 age = 3;
+	string IP = 4;
+	string nominatedNodeName = 5;
+	string nodeName = 6;
+	int32 restartCount = 7;
+	repeated ContainerStatus containerStatuses = 8;
+	map<string, string> labels = 9;
+	map<string, string> annotations = 10;
+	repeated OwnerReference ownerReferences = 11;
+	repeated string tags = 12;
+	bytes yaml = 13;
+}
+
+message ContainerStatus {
+	string name = 1;
+	string containerID = 2;
+	bool ready = 3;
+	bool started = 4;
+}
+
+message OwnerReference {
+	string apiVersion = 5;
```

Comment:
nit, if we're intentionally omitting field `2`, can we mark it as `reserved`?

Reply:
weird, it's copy pasted from https://github.com/kubernetes/kubernetes/blob/006378fcc1c8db4f7063812b67a59969fc0d09f0/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto#L699 I guess I can reorder the ids